<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMode</class>
 <widget class="QWidget" name="UserMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Mode</string>
  </property>
  <widget class="QListWidget" name="ItemDisplay">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="NextButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>160</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="SaveButton">
      <property name="text">
       <string>Save Organic Fragment</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DisplayButton">
      <property name="text">
       <string>Display Mylist</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="FilterButton">
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OpenButton">
      <property name="text">
       <string>Open Mylist File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="UndoButton">
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RedoButton">
      <property name="text">
       <string>Redo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
